front/selector.js

```diff
@@ -0,0 +1,13 @@
+export default function select(data, getList, defaults = {}) {
+  if (data && typeof data === 'object') {
+    const selected = {}
+    for (const d in data) {
+      if (getList.includes(d)) {
+        selected[d] = data[d]
+      } else if (defaults[d]) {
```

Comment:
Meio que desnecessário porque se for undefined ele vai atribuir undefined, dá no mesmo sem o else if.

Reply:
Verdade, tirado.